client/src/actions/citiesActions.js

```diff
@@ -0,0 +1,63 @@
+import axios from "axios"
+import { GET_CITIES, ADD_CITY, CITIES_LOADING, ADD_SUCCESS, DELETE_CITY, UPDATE_CITY } from "./types"
+import { tokenConfig } from "./authActions"
+
+
+export const getCities = () => dispatch => {
```

Comment:
Looking through the actions for interacting with the RESTful back end I think it makes sense to come up with a common naming scheme for the methods.  Something just make things very obvious what CRUD action is happening to make things easy to quickly read.  E.g. `saveUserDetails` and `saveComment` for POST actions.